packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -33,18 +34,39 @@ export const MenuList = compose<MenuListType>({
   ...stylingSettings,
   slots: {
     root: MenuStack,
+    focusZone: FocusZone,
   },
   useRender: (userProps: MenuListProps, useSlots: UseSlots<MenuListType>) => {
     const menuList = useMenuList(userProps);
     const contextValue = useMenuListContextValue(menuList);
     const Slots = useSlots(menuList.props);
 
+    const focusZoneRef = React.useRef<View>();
+
+    React.useEffect(() => {
+      focusZoneRef?.current?.focus();
+    }, []);
+
     return (_final: MenuListProps, children: React.ReactNode) => {
-      return (
-        <MenuListProvider value={contextValue}>
+      const content =
+        Platform.OS === 'macos' ? (
+          <Slots.root>
+            <FocusZone
```

Comment:
Should this be Slots.focusZone?

Also nit: I wonder if you could define a wrapper component that's FocusZone on one platform and React.fragment on a different one (or swap the slot per platform?). Then the render code would look the same on all plats:

```
<Slots.root>
    <WrapperComponent>
        {children}
    </WrapperComponent>
</Slots.root>
```

Reply:
Right, good catch. 
A wrapper component would probably work. iirc, we've done this elsewhere. I'll look into it

Reply:
Actually, I could make a follow-up change to let componentRef also be of type 'React.ForwardedRef' in all our components I suppose.

Reply:
The wrapper thing was more of a suggestion, it seems like it's not worth the work, which is fine.

Reply:
Filed https://github.com/microsoft/fluentui-react-native/issues/2283 about the componentRef forwarded ref thing. For this PR, I think I'll keep it as is. I also worry that not using `stagedComponent` for FocusZoneWrapper might be bad.. even though I'm pretty sure I can't since I need the ref.